configure.ac

```diff
@@ -28,6 +28,12 @@ BITCOIN_GUI_NAME=bitcoin-qt
 BITCOIN_CLI_NAME=bitcoin-cli
 BITCOIN_TX_NAME=bitcoin-tx
 
+dnl Unless the user specified ARFLAGS, force it to be cr
+AC_ARG_VAR(ARFLAGS, [Flags for the archiver, defaults to <cr> if not set])
```

Comment:
Nit: I'd use `"cr"` or so as `<cr>` looks like the usual writing of 'carriage return'. Otherwise, looks fine to me.